### PR TITLE
chore: converted withdraw and ragequit with tx legos

### PR DESCRIPTION
**Pull request by sunguru98**
My first attempt along with @jordanlesich in converting the withdraw and ragequit submit forms in accordance to txLegos :)

### CODE REVIEW

**Comment by scottrepreneur** [comment]
Task linked: [CU-y3jwqc Transaction/Form Refactor](https://app.clickup.com/t/y3jwqc)

**Review by scottrepreneur — APPROVED** [review]

File: src/data/contractTX.js [hunk 1]
```diff
@@ -136,4 +137,22 @@ export const TX = {
...
+  WITHDRAW_PROPOSAL: {
+    contract: 'Moloch',
+    name: 'withdrawBalance',
+    poll: 'subgraph',
+    onTxHash: ACTIONS.BASIC,
+    display: 'Withdraw Balance',
+    errMsg: 'Error Withdrawing Balance',
+    successMsg: 'Balance Withdrawn!',
+  },
+  RAGE_QUIT_PROPOSAL: {
```
Comment (skuhlmann) [inline src/data/contractTX.js]: this name is confusing - it's not a proposal - so let's just use 'RAGE_QUIT'

Comment (sunguru98) [reply src/data/contractTX.js]: Oh yeah .. my bad .. Will change it :)

**Comment by sunguru98** [comment]
Donezo @skuhlmann

**Review by scottrepreneur — APPROVED** [review]

**Pull request merged**